src/main/joysoundApi.ts

```diff
@@ -163,13 +166,10 @@ export class JoysoundAPI extends RESTDataSource {
   }
 
   async getMovieUrls(id: string) {
-    const creds = await this.credsProvider();
-
-    return this.get<JoysoundMovieList>(
-      "/player/contentsInfo",
-      { songNumber: id, serviceType: "003000761" },
-      { headers: { Cookie: generateCookieString(creds.cookies) } }
-    );
+    return this.post<JoysoundMovieList>("/player/contentsInfo", {
```

Comment:
Can you just cherry pick this file's changes and make a new PR? This would need fixing anyway eventually. Thanks for doing it!